packages/brand-moment/src/stories/BrandMoment.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react"
+import {
+  EmptyStatesAction,
+  EmptyStatesNegative,
+  EmptyStatesNeutral,
+  EmptyStatesPositive,
+} from "@kaizen/draft-illustration"
+import arrowRightIcon from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import securityTipIcon from "@kaizen/component-library/icons/security-tip.icon.svg"
+import mailIcon from "@kaizen/component-library/icons/email.icon.svg"
+import feedbackClassifyIcon from "@kaizen/component-library/icons/feedback-classify.icon.svg"
+import { Box, Paragraph } from "@kaizen/component-library"
+import { BrandMoment } from "../../"
+import {
+  MinimalBasic,
+  MinimalCustomerFocused,
+  FakeNavBar,
+} from "./ExampleHeaders"
+
+export default {
+  title: "Brand Moment (React)",
+  component: BrandMoment,
+  parameters: {
+    info: {
+      text: `
+        import { BrandMoment } from "@kaizen/brand-moment";
+      `,
+    },
+  },
+  decorators: [story => <div style={{ margin: "-1rem" }}>{story()}</div>],
+}
+
+export const DemoIntro = () => (
+  <BrandMoment
+    backgroundColor="blue"
+    header={<MinimalBasic />}
+    illustration={<EmptyStatesAction alt="" />}
+    subheading="Welcome to Culture Amp"
+    heading="Let's dive in and see how it works"
```

Comment:
This is extreme nit-pick territory but since I can see these being copied around the place a bit, we should ideally use proper apostrophes in all the copy for these examples:

```diff
- Let's dive in and see how it works
+ Let’s dive in and see how it works
```

Reply:
Good call